### DISCUSSION TITLE
Change id selection way

### DISCUSSION THREAD

**Discussion by fjtrooper** [PostgREST/postgrest#1966] [Q&A] [answered]
Hi! first of all congratulations for postgrest, I think it's a very good project!
I'd like to ask you if it's possible to change the way postgrest identify record id, I mean use /products/1  instead /products?id=1

Thanks!!

**Comment by steve-chavez** [comment, marked as answer]
Hey @fjtrooper,

Thanks! Right now this can be done through Nginx, you can see how to do it [here](https://postgrest.org/en/v8.0/admin.html#alternate-url-structure).

We also have plans to allow this on PostgREST itself, check https://github.com/PostgREST/postgrest/issues/1909.

**Reply by fjtrooper** [reply]
ok thanks I'll check!
best regards!
